docs/CUSTOMIZING.md

```diff
@@ -0,0 +1,84 @@
+# Customization guide
+
+This document describes how you can create your own app for an event.
+
+## 1. Required information
+
+The following information are needed to configure the app for your event.
+
+- Schedule URL which provides Frab compatible XML
+- Session URL template, e.g. `https://awesome-event.com/2021/events/%1$s.html`
+- Server backend type, one of: `pentabarf`, `frab`, `pretalx`
+- Schedule feedback URL template (optional), e.g. `https://awesome-event.com/2021/events/%s/feedback/new`
```

Comment:
I would recommend to make first all necessary ones, and later the optional ones.

Reply:
Thanks. :heavy_check_mark: Moved down in 4d069023cf26b8e4089bad08b5d1fddb2949727c.